src/cbc_sdk/platform/alerts.py

```diff
@@ -1033,6 +1036,29 @@ def set_time_range(self, *args, **kwargs):
             self._time_range = time_filter
         return self
 
+    def _get_query_parameters(self):
```

Comment:
Why can't we use `_build_request` and modify the request if needed after? I don't want to maintain multiple ways to create the search request